_includes/sidemenu_worker.html

```diff
@@ -21,6 +21,7 @@
   <li><a {% if page.url == "/worker/reference/payload/index.html" %}class="selected" {% endif %}href="/worker/reference/payload" title="Payload">Payload Retrieval</a></li>
   <li><a {% if page.url == "/worker/reference/security/index.html" %}class="selected" {% endif %}href="/worker/reference/security" title="Security">IronWorker Security</a></li>
   <li><a {% if page.url == "/worker/scheduling/index.html" %}class="selected" {% endif %}href="/worker/scheduling" title="Scheduling">Scheduling</a></li>
+  <li><a {% if page.url == "/worker/hybrid/index.html" %}class="selected" {% endif %}href="/worker/hybrid" title="Hybrid Workers">Scheduling</a></li>
```

Comment:
@rdallman Maybe `<... title="Hybrid Workers">Hybrid Workers</a>` instead of `<... title="Hybrid Workers">Scheduling</a>`?